main/java/com/epam/utils/Utils.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.utils;
+
+import com.epam.dao.EntityDao;
+import com.epam.entity.UserAccount;
+import com.epam.exceptions.BusinessException;
+import org.apache.log4j.Logger;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+
+/**
+ * Created by Almas_Doskozhin
+ * on 2/21/2016.
+ */
+public class Utils {
+    private static final Logger LOG = Logger.getLogger(Utils.class);
+
+    private Utils(){}
+
+    public static boolean checkUsers(){
+        try {
+            EntityDao entityDao = new EntityDao();
+            UserAccount user1 = entityDao.readUserAccount("user1");
+            UserAccount user2 = entityDao.readUserAccount("user2");
+
+            boolean user1Correct = user1.getDollarAmount().intValue()==30 &&
```

Comment:
You repeat yourselve, cause you call same checks for two different users.
Could you move logic into UserAccount class and call new methods for each user?